src/cli.coffee

```diff
@@ -35,6 +35,7 @@ watchCmd = program
   .option('-p, --port [port]', 'if a `server` option was specified, define on which port
  the server would run')
   .option('-d, --debug', 'print verbose debug output to stdout')
+  .option('-l, --stdin', 'listen to stdin and shut down when stdin closes')
```

Comment:
I decided to not use "-" because it usually means you can write stuff in
the stdin and it will be read. Here we just care about EOF.



-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*